### PR TITLE
Javascript-light-bulb exercise - Ashley Ho

### PR DESCRIPTION
**Pull request by ashleylho**
https://user-images.githubusercontent.com/60860374/188758876-05b35c5f-548d-4dac-9c15-4a43062247d8.mov

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+var $body = document.querySelector('body');
+var $circle = document.querySelector('.circle');
+
+function handleClick(event) {
+  if ($body.className === 'lightbulb') {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:5]: By doing this, your logic is dependent on the current state of the DOM, which is not ideal. Instead, I would create a variable to keep track of if the light is currently on or not, and use that variable's value to determine what action to take when the light is clicked.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ashleylho** [comment]
Screen Shot 2022-09-07 at 1 17 35 PM

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -2,16 +2,14 @@ var $body = document.querySelector('body');
 var $circle = document.querySelector('.circle');
 
 function handleClick(event) {
-  if ($body.className === 'lightbulb') {
+  var currentStatus = $body.className;
+  if (currentStatus === 'lightbulb') {
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:6]: This is the same problem but now in a new variable 🤩

Comment (codingcodymiller) [reply javascript-light-bulb/main.js:6]: You should make a variable that has a boolean value to keep track of whether the light is on or not and use that variable to determine what action to take each time the lightbulb is clicked.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 2 above)
Comment (ashleylho) [reply javascript-light-bulb/main.js:6]: > This is the same problem but now in a new variable 🤩

My bad.... 😅

**Comment by ashleylho** [comment]
Screen Shot 2022-09-07 at 8 06 15 PM

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**